src/components/SearchBar.css

```diff
@@ -0,0 +1,31 @@
+.search {
+    position: relative;
+	border: 2px solid #3fc5cf;
+	overflow: auto;
+	border-radius: 35px;
+    margin-top: 3%;
+    margin-left: 2%;
+    text-align: center;
+    width: 30%;
+    opacity: 0.5;
+}
+
+.bar {
+    border:0;
+    text-align: center;
+    width: 85%;
+}
+
+.search:hover {
+    opacity: 1.0;
+    transition-duration: 0.5s;
+}
+
+.search-icon {
+    position: relative;
+    padding-left: 2%;
+}
+
+*:focus {
+    outline: none;
+}
```

Comment:
looks like GitHub doesn't like it when you don't have a new line at the end of the file.